contracts/tribute/src/msg.rs

```diff
@@ -27,4 +29,6 @@ pub enum ExecuteMsg {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
-pub struct MigrateMsg {}
+pub struct MigrateMsg {
+    pub min_prop_percent_to_deploy: Uint128,
```

Comment:
Maybe we can also leave out the deploying part here and just make it about refunding vs claiming tributes. something like "min_prop_percent_for_claimable_tributes"